<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>786</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>761</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>533</width>
      <height>0</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>989</width>
       <height>759</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>MovideXPilot</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>509</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLineEdit" name="FilmInput">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 40px;
width: 40px;s
}</string>
         </property>
         <property name="placeholderText">
          <string>Filme Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="leerenButton">
            <property name="text">
             <string>Suche Leeren</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="erweitereSucheCheckBox">
            <property name="text">
             <string>Erweiterte Suche</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="FilmInputLine">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 20px;
width: 20px;s
}</string>
               </property>
               <property name="placeholderText">
                <string>Schauspieler</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="FilmInputLine_4">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 20px;
width: 20px;s
}</string>
               </property>
               <property name="placeholderText">
                <string>Schauspieler</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="FilmInputLine_7">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 20px;
width: 20px;s
}</string>
               </property>
               <property name="placeholderText">
                <string>Schauspieler</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="FilmInputLine_6">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 20px;
width: 20px;s
}</string>
               </property>
               <property name="placeholderText">
                <string>Schauspieler</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="FilmInputLine_5">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 20px;
width: 20px;s
}</string>
               </property>
               <property name="placeholderText">
                <string>Schauspieler</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="FilmInputLine_8">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
background-color:rgb(34,36,44);
height: 20px;
width: 20px;s
}</string>
               </property>
               <property name="placeholderText">
                <string>Schauspieler</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sucheButton">
         <property name="text">
          <string>suche</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>frame_4</zorder>
      <zorder>sucheButton</zorder>
      <zorder>frame_11</zorder>
      <zorder>FilmInput</zorder>
      <zorder>listWidget</zorder>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-30</y>
        <width>991</width>
        <height>80</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:rgb(25, 63, 71);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="verlassenButton">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>40</y>
         <width>80</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>verlassen</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="OutputName">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>691</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;s
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="OutputBeschreibung">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>841</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
border: 2px solid rgb(37,39,48);
border-radius: 20px;
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;s
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Beschreibung:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Produzent:</string>
      </property>
     </widget>
     <widget class="QLabel" name="OutputProduzent">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>510</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-left: 20px;
padding-right:20px;
height:60px;
width: 60px;s
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>560</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Dauer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="OutputDauer">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>560</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;s
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>600</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Erscheinungsjahr:</string>
      </property>
     </widget>
     <widget class="QLabel" name="OutputErscheinungsjahr">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>600</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;s
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>frame_8</zorder>
     <zorder>label</zorder>
     <zorder>OutputName</zorder>
     <zorder>OutputBeschreibung</zorder>
     <zorder>label_4</zorder>
     <zorder>label_6</zorder>
     <zorder>OutputProduzent</zorder>
     <zorder>label_2</zorder>
     <zorder>OutputDauer</zorder>
     <zorder>label_5</zorder>
     <zorder>OutputErscheinungsjahr</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verlassenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
